gui/widgets/grid.cpp

```diff
@@ -40,8 +40,15 @@ GridItemWidget::GridItemWidget(GridWidget *boss, int x, int y, int w, int h)
 	_grid = boss;
 	isHighlighted = false;
 }
+
 GridItemWidget::GridItemWidget(GridWidget *boss)
-	: GridItemWidget(boss, 0, 0, 0, 0) {}
+	: ContainerWidget(boss, 0, 0, 0, 0), CommandSender(boss) {
+
+	setFlags(WIDGET_ENABLED | WIDGET_TRACK_MOUSE | WIDGET_CLEARBG);
+	_activeEntry = nullptr;
+	_grid = boss;
+	isHighlighted = false;
```

Comment:
This member should start with an underscore.

Reply:
Thanks, fixed in f851cea.